docker/creds/credentials_test.go

```diff
@@ -643,20 +644,33 @@ func correctVerifyCbk(ctx context.Context, image string, credentials Credentials
 	return creds.ErrUnauthorized
 }
 
-func withCleanHome(t *testing.T) {
-	t.Helper()
-	homeName := "HOME"
+func testHomeEnvName(t *testing.T) string {
```

Comment:
Why does this take `*testing.T`?

Reply:
For the `t.Helper(t)` call which was missing... and now is not.